public/manifest.json

```
@@ -26,5 +26,18 @@
       }
     }
   ],
-  "capture_links": "new_client"
+  "capture_links": "new_client",
+  "share_target": {
+    "action": "/",
+    "method": "POST",
+    "enctype": "multipart/form-data",
+    "params": {
+      "files": [
+        {
+          "name": "file",
+          "accept": ["application/json", ".excalidraw"]
```

```suggestion
          "accept": [".excalidraw"]
```

Let’s be very conservative here and strictly only accept the file extension.

---

I was testing it on my android phone with Chrome 89, it's not showing Excalidraw in share sheet when trying to share `.excalidraw` files. It works with JSON. Weird!

---

We were trying to mint our own media type, but it didn’t work with Safari. For now triggering for all .json files should work, but in the future I should investigate again if we can use something more specific.

---

Got it! I added `application/json` back 👍🏽

---

Let’s add `application/vnd.excalidraw+json`, too, this is what we were trying to use. It’s also in the manifest a couple of lines up.

---

Done 👍🏽